tests/TestClassDatabaseChanges.php

```diff
@@ -138,7 +138,7 @@ public function testInstallMigrationsDataSimilarEstates(): array
 		$this->assertEquals('Field 1', $newFields[0]);
 		$this->assertEquals('Field 2', $newFields[1]);
 		$this->assertEquals('Field 3', $newFields[2]);
-		$this->assertTrue(true, (string)$newRadius);
+		$this->assertEquals(35, $newRadius);
 		$this->assertTrue(true, (string)$newSameKind);
```

Comment:
there are two more of these

Reply:
I updated it

Reply:
What exactly is the `(string)` cast there for? If the usage of `assertTrue` was intended, the first parameter would have to be removed (`true`).

Reply:
ok I will update it